### DISCUSSION TITLE
Feature Suggestion：Add network proxy option to weather widget

### DISCUSSION THREAD

**Discussion by AelepantB** [valinet/ExplorerPatcher#2943] [Ideas]
I love this app. It solved a problem for me. Win11 does not let me adjust the taskbar height or change the taskbar merging function. I tried many ways to make the Win11 taskbar behave like before, but none of them worked. Your app fixed this issue for me.

The weather widget is really great. I don't have to open the browser or my phone to check the weather forecast. Sometimes, looking out of the window is not very reliable.

My suggestion: Add a proxy option in the weather widget. This would allow users who care about their privacy or have poor connections to choose their preferred network.

Thank you for creating such a wonderful product.

**Comment by pyrates999** [comment]
The weather widget should use whatever proxy is being set in the settings application that windows provides.

**Reply by AelepantB** [reply]
Windows has a built-in system proxy, but I use different proxies for different applications, so I don't turn on the system proxy. ExplorerPatcher should also have proxy options for its updates.
